File: tmp-mei/fgen4255027576711118260/base/flows/ActorRes_0.java
package base.flows;
public interface ActorRes_0 extends base.Sealed_0{
ActorRes_0 $self = new ActorRes_0Impl();
Object match$imm(base.flows.ActorResMatch_1 m_m$);

base.flows.ActorRes_0 stop$imm();

base.flows.ActorRes_0 continue$imm();
static Object match$imm$fun(base.flows.ActorResMatch_1 m_m$, base.flows.ActorRes_0 $this) {
  return (switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (Object) null;
}})
;
}

static base.flows.ActorRes_0 continue$imm$fun(base.flows.ActorRes_0 $this) {
  return ((base.flows.ActorRes_0)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.flows.ActorRes_0) null;
}})
);
}

static base.flows.ActorRes_0 stop$imm$fun(base.flows.ActorRes_0 $this) {
  return ((base.flows.ActorRes_0)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.flows.ActorRes_0) null;
}})
);
}
}

File: tmp-mei/fgenjs5030319945734616766/base/flows/ActorRes_0.js
import { base$$Sealed_0 } from "../../base/Sealed_0.js";
import * as rt from "../rt/main.js";

export class base$$flows$$ActorRes_0 extends base$$Sealed_0 {
  static $self = new base$$flows$$ActorRes_0();

  match$imm(m_m$) {
  return base$$flows$$ActorRes_0.match$imm$fun(m_m$, this);
}


stop$imm() {
  return base$$flows$$ActorRes_0.stop$imm$fun(this);
}


continue$imm() {
  return base$$flows$$ActorRes_0.continue$imm$fun(this);
}

  static match$imm$fun(m_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

    static continue$imm$fun($this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

    static stop$imm$fun($this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

}
